pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -136,6 +138,16 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 		if notBestEffort {
 			return lifecycle.PodAdmitResult{Admit: true}
 		}
+
+		// Admit BestEffort Pod if they tolerates memory pressure; dependents on eviction loop to
```

Comment:
If it's high priority pod, evict it by sync loop instead of rejecting here.

Reply:
Oh, I get it.  You are saying pods may be evicted since the node is under memory pressure.  I don't think the comment about the eviction loop is necessary, as pods may not be evicted at all (if they are using a soft eviction threshold, for example)

Reply:
Oh, I get it.  You are saying pods may be evicted since the node is under memory pressure.  I don't think the comment about the eviction loop is necessary, as pods may not be evicted at all (if they are using a soft eviction threshold, for example)

Reply:
+1 to David's comment.

Reply:
done